dotnet/src/SemanticKernel.UnitTests/Functions/KernelParameterMetadataTests.cs

```diff
@@ -140,7 +141,9 @@ public void ItInvalidatesSchemaForNewDefaultValue()
     internal sealed class Example
```

Comment:
Type-level annotations should be supported as well, in case you wanted to test for that.

Reply:
Thanks